src/components/Breadcrumb.vue

```diff
@@ -0,0 +1,100 @@
+<script>
+export default {
+  props: {
+    location: {
+      type: Object,
+      required: true,
+      validator: val => val.type && val.id,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    append: {
+      type: Array,
+      default: () => [],
+    },
+  },
+  inject: ['girderRest'],
+  data() {
+    return {
+      loading: false,
+    };
+  },
+  asyncComputed: {
+    breadcrumb: {
+      default: { root: {}, path: [] },
+      async get() {
+        this.loading = true;
+        const breadcrumb = { root: {}, path: [] };
+        const { id, type, name } = this.location;
+        if (type === 'folder') {
+          // The last breadcrumb isn't returned by rootpath.
+          if (name) {
+            breadcrumb.path.unshift(this.location);
+          } else {
+            const { data } = await this.girderRest.get(`folder/${id}`);
+            breadcrumb.path.unshift(this.extractCrumbData(data));
+          }
+          // Get the rest of the path.
+          const { data } = await this.girderRest.get(`folder/${id}/rootpath`);
+          data.reverse().forEach((crumb) => {
+            const { object } = crumb;
+            const entity = this.extractCrumbData(object);
+            if (entity.type === 'folder') {
+              breadcrumb.path.unshift(entity);
+            } else {
+              breadcrumb.root = entity;
+            }
+          });
+        } else {
+          const { data } = await this.girderRest.get(`${type}/${id}`);
+          breadcrumb.root = this.extractCrumbData(data);
+        }
+        this.loading = false;
+        return breadcrumb;
+      },
+    },
+  },
+  methods: {
+    extractCrumbData(object) {
+      return {
+        name: object._modelType !== 'user' ? object.name : object.login,
+        id: object._id,
+        type: object._modelType,
+        parentId: object.parentId,
+        parentType: object.parentCollection,
+      };
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+v-breadcrumbs.girder-breadcrumb-component.font-weight-bold.pa-0
+  v-icon.mdi-24px(
+      :disabled="disabled",
+      slot="divider",
+      color="accent") {{ $vuetify.icons.chevron }}
+  v-breadcrumbs-item(
+      :disabled="disabled || breadcrumb.path.length === 0",
+      @click.native="$emit('crumbclick', breadcrumb.root)")
+    v-icon.mdi-24px.pr-2(color="accent") {{ $vuetify.icons.globe }}
+    | {{ breadcrumb.root.name }}
+  v-breadcrumbs-item(
+      :disabled="disabled || index == breadcrumb.path.length-1",
+      v-for="(item, index) in breadcrumb.path",
+      :key="item.id",
+      @click.native="$emit('crumbclick', item)") {{ item.name }}
+  v-breadcrumbs-item(
+      v-for="item in append",
+      :key="item.id") {{ item }}
+</template>
+
+<style lang="scss">
+.girder-breadcrumb-component {
+  &.v-breadcrumbs li:nth-child(2n) {
+    padding: 0 4px !important;
```

Comment:
I'm still in favor of removing this rule, I don't think the padding is important enough for us to be reaching deep into Vuetify internals to change it. I think our best bet would be to open an issue request (or even a PR!) against Vuetify to allow a `dense` prop on the breadcrumb component, or make it so that the `divider` slot is in charge of its own padding (which may be harder to pitch to them since it's breaking).

Reply:
I'd prefer not to have this as well, but I find the spacing between dividers (12px on either side) to be unreasonably large.  The actual problem isn't the padding -- it's the mdi-chevron icon which makes really poor use of its vector box - it has another 12px between the wall of the image box and the start of the icon, making the gap look like ~24px to either side.

I don't think this is a large enough use case for the Vuetify maintainers to accept a `dense` prop as compensation for an abnormal icon, and fixing the icon doesn't seem like a great option either.

Reply:
What about if we just use a `/` or a [unicode chevron](https://www.compart.com/en/unicode/U+203A)?

Reply:
I like the slash too, and it also will be familiarly associated with the filesystem separator.